elasticsearch716/build/setup_docker.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -x
+set -e
+
+# setup
+apt-get update
+apt-get install -y --no-install-recommends ca-certificates-java
+
+ES_PKG_NAME=elasticsearch-7.16.3
+arch="$([ "`uname -m`" = "aarch64" ] && echo "aarch64" || echo "x86_64")"
+
+cd /tmp/
+wget -nv -t5 -O es.tar.gz https://artifacts.elastic.co/downloads/elasticsearch/$ES_PKG_NAME-linux-$arch.tar.gz
```

Comment:
why not?
wget -nv -t5 -O es.tar.gz "https://artifacts.elastic.co/downloads/elasticsearch/$ES_PKG_NAME-linux-$(uname -m).tar.gz"